chain/ethereum/src/network_indexer/ethereum.graphql

```diff
@@ -62,25 +62,126 @@ type Block @entity {
+  """
+  A list of ommer (AKA uncle) blocks associated with this block.
+  """
   ommers: [Block]!
 
   """The keccak256 hash of all the ommers (AKA uncles) associated with this block."""
   ommerHash: Bytes!
 
-  # """
-  # A list of transactions associated with this block.
-  # """
-  # transactions: [Transaction!]
+  """
+  A list of transactions associated with this block.
+  """
+  transactions: [Transaction!]
+
+  """The number of logs emitted in this block."""
+  logCount: Int!
 
-  # """The logs emitted in this block."""
-  # logs: [Log!]!
+  """The logs emitted in this block."""
+  logs: [Log!]
 
   """Size of the block in bytes."""
   size: BigInt
 
   """Seal fields."""
   sealFields: [Bytes!]!
 }
+
+""" An Ethereum transaction. """
+type Transaction @entity {
+  id: ID!
+
+  """ The hash of this transaction. """
+  hash: Bytes!
+
+  """ The nonce of the account this transaction was generated with. """
+  nonce: BigInt!
+
+  """
+  The index of this transaction in the parent block.
+  This will be null if the transaction has not yet been mined.
+  """
+  index: BigInt
+
+  """ The account that sent this transaction - this will always be an externally owned account. """
+  from: Bytes!
+
+  """ The account the transaction was sent to. This is null for contract-creating transactions. """
+  to: Bytes
+
+  """ The value, in wei, sent along with this transaction. """
+  value: BigInt!
+
+  """ The price offered to miners for gas, in wei per unit. """
+  gasPrice: BigInt!
+
+  """ The maximum amount of gas this transaction can consume. """
+  gas: BigInt!
+
+  #  """ The data supplied to the target of the transaction. """
+  #  inputData: Bytes!
+
+  """
+  The block this transaction was mined in.
+  This will be null if the transaction has not yet been mined.
+  """
+  block: Block
+
+  """ The number of the block this transaction was mined in. """
+  blockNumber: Int!
+
+  """The number of logs emitted in this transaction."""
+  logCount: Int!
+
+  """
+  A list of log entries emitted by this transaction.
+  If the transaction has not yet been mined, this field will be null.
+  """
+  logs: [Log!]
+
+  """
+  The return status of the transaction.
+  This will be 1 if the transaction succeeded,
+  or 0 if it failed (due to a revert, or due to running out of gas).
+  If the transaction has not yet been mined, this field will be null.
+  """
+  status: BigInt
+
+  """
+  The amount of gas that was used processing this transaction.
+  If the transaction has not yet been mined, this field will be null.
+  """
+  gasUsed: BigInt
+
+  """
+  The total gas used in the block up to and including this transaction.
+  If the transaction has not yet been mined, this field will be null.
+  """
+  cumulativeGasUsed: BigInt
+
+  """
+  The account that was created by a contract creation transaction.
+  If the transaction was not a contract creation transaction, or it has not yet been mined, this field will be null.
+  """
+  createdContract: Bytes
+}
```

Comment:
I see that Geth also includes
```graphql
r: BigInt!
s: BigInt!
v: BigInt!
```

But I agree that we don't need this information for now; besides, it's not directly available from the rust-web3 types.